webpack.base.js

```diff
@@ -180,6 +181,11 @@ const config = {
       process: 'process/browser',
       Buffer: ['buffer', 'Buffer'],
     }),
+    new InjectManifest({
+      swSrc: './src/workers/firebase-messaging-sw.ts',
+      swDest: 'firebase-messaging-sw.js',
+      exclude: [/./],
+    }),
```

Comment:
So we will need to add one of these for each service worker we create?

Reply:
indeed however there is a different version of creating a web worker with the google web work called `generateSW` but that method doesnt work with the notifications api

https://developer.chrome.com/docs/workbox/the-ways-of-workbox/